examples/custom-elements/index.html

```diff
@@ -2,6 +2,6 @@
     <head>
     </head>
     <body>
-        <script type="module" src="./errors.js"></script>
+        <script type="module" src="./bundle.js"></script>
```

Comment:
I moved all examples inside their own folder and each one now loads a  'bundle.js' file that is built.